runner/ansible/roles/1.3.6/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Set Test ID
```

Comment:
This test must be revisited.
Check the original one:
https://github.com/trento-project/trento/blob/main/examples/generic-azure.yaml

This check doesn't have an expected value. It checks if the `wdtimeout` has the double value of `msgwait`.
Maybe the test was updated since you implemented this, but it looks really different.
**In fact, it looks like you implemented here the 1.3.5 test*

Reply:
it needs the following:
DEF_MSGWAIT={{ expected[test_id] }}

expected 1.3.5 is 60 and   expected for 1.3.6 is 120  but it look like 1.3.6 is always checking msgwait and 1.3.5 watchdog.